test/validate-examples.mjs

```diff
@@ -11,30 +11,34 @@ addFormats(ajv);
 const errors = [];
 const __filename = fileURLToPath(import.meta.url);
 
-const examples = await glob(
-  path.join(path.dirname(__filename), "/../domains/**/examples/*.json")
-);
-
-examples.forEach((examplePath) => {
-  const schemaPath = path.join(
-    path.dirname(path.dirname(examplePath)),
-    "schema.json"
-  );
+const schemas = await glob(
```

Comment:
Looping over examples meant we tried to validate schemas twice if more than one example per schema existed. This way we can add multiple examples which is (sometimes) useful.